src/main/resources/application.properties

```diff
@@ -5,15 +5,15 @@ spring.jackson.default-property-inclusion=non_null
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
 build.version=
 
-server.port=8080
+server.port=8089
```

Comment:
Please do not change the main application.properties settings since it will ruin for everyone using it.

Reply:
OK